modules/ipi-install-configure-multiple-cluster-nodes.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_bare_metal_ipi/ipi-install-installation-workflow.adoc
+
+:_content-type: CONCEPT
+[id="ipi-install-configure-multiple-cluster-nodes_{context}"]
+= Configure multiple cluster nodes
+
+You can simultaneously configure {product-title} cluster nodes with identical settings. Configuring multiple cluster nodes avoids adding redundant information for each node to the `install-config.yaml` file. This file contains specific parameters to apply identical configuration to multiple nodes in the cluster.
```

Comment:
```suggestion
You can simultaneously configure {product-title} cluster nodes with identical settings. Configuring multiple cluster nodes avoids adding redundant information for each node to the `install-config.yaml` file. This file contains specific parameters to apply the same configuration to multiple nodes in the cluster.
```
Or:
"to apply an identical configuration to multiple nodes"
"to apply identical configurations to multiple nodes"